backend/src/tasks/price-updater.ts

```diff
@@ -103,13 +103,13 @@ class PriceUpdater {
   /**
    * Fetch last BTC price from exchanges, average them, and save it in the database once every hour
    */
-  private async $updatePrice(): Promise<void> {
+  private async $updatePrice(storeInDb: boolean): Promise<void> {
     if (this.lastRun === 0 && config.DATABASE.ENABLED === true) {
       this.lastRun = await PricesRepository.$getLatestPriceTime();
     }
 
-    if ((Math.round(new Date().getTime() / 1000) - this.lastRun) < 3600) {
-      // Refresh only once every hour
+    if ((Math.round(new Date().getTime() / 1000) - this.lastRun) < Math.min(config.MEMPOOL.PRICE_UPDATE_FREQUENCY, 3600)) {
```

Comment:
If you make the refresh time configurable, don't cap the value. Maybe I only want to refresh prices once a day 👍🏻